submissions/famuyiwaOlawumi/htmltask/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```

Comment:
Did you decide to rework your product page again

Reply:
You already have an `index.html` file in HTML folder. Is there a reason why you are creating this again?

Reply:
@Olawumi11 Also, remove the images associated with this file after deleting this page